### ISSUE TITLE
design remote-asr plugin

### ISSUE THREAD

**Issue by timobaumann** [dialogos-project/dialogos#210]
It would be nice to have remote-ASR nodes and a plugin that is able to manage several alternative ASR services (such as Google, AWS, Kaldi, ...). Although these would not support grammars, they will be useful, e.g. for free-form input as a calendar entry title/content. 

For Google's ASR, see: https://cloud.google.com/text-to-speech/docs/reference/libraries#client-libraries-install-java

**Comment by alexanderkoller** [comment]
Yes, I agree. Before we make major changes to the ASR architecture, though, let's talk this through carefully; I've toyed with similar questions for the Alexa plugin.

**Comment by timobaumann** [comment]
see also https://github.com/mischa99/dialogos-plugin-GoogleASR
